### PR TITLE
Abstract notification log

### PR DESCRIPTION
**Pull request by OmeGak**

### CODE REVIEW

File: indico/htdocs/css/jquery-ui.css [hunk 1]
```diff
@@ -791,7 +791,6 @@ button.ui-button::-moz-focus-inner {
 .ui-widget-content {
 	border: 1px solid #a6c9e2;
 	background: #fcfdfd url(../images/jquery-ui/ui-bg_inset-hard_100_fcfdfd_1x100.png) 50% bottom repeat-x;
-	color: #222222;
```
Comment (ThiefMaster) [inline indico/htdocs/css/jquery-ui.css]: What's the reason for this change? I think this is a mostly generated file...

File: indico/modules/events/abstracts/templates/abstract/notification_log.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            </div>
+            <time datetime="{{ entry.sent_dt.isoformat() }}">
+                {{- entry.sent_dt|format_human_date -}}
+            </time>
+        </div>
+        {% if not legacy %}
+            <div class="content">
+                <div class="metadata">
+                    <div class="flex">
+                        {%- trans recipients=entry.recipients|join(', ') -%}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/abstract/notification_log.html]: `|sort`?

File: indico/modules/events/abstracts/templates/abstract/notification_log.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+<div id="abstract-notification-log">
+    {% for log_entry in abstract.email_logs|reverse %}
+        {{ render_entry(log_entry) }}
+    {% endfor %}
+</div>
+
+<script>
+    $('#abstract-notification-log .metadata:not(.legacy)').on('click', function() {
+        var $this = $(this);
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/abstract/notification_log.html]: `$(this).toggleClass('open').siblings('.content').slideToggle();`?

Quite short so it's still pretty readable in a single line. No strong opinion though.

File: indico/htdocs/css/jquery-ui.css (hunk 1 above)
Comment (OmeGak) [inline indico/htdocs/css/jquery-ui.css]: Rats. I didn't think it was a generated file. It's safe to undo this change since this should override it: https://github.com/indico/indico/pull/2668/files#diff-cd5654bb800f0db30dc920d3492c2cccR19

File: indico/modules/events/abstracts/templates/abstract/notification_log.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            <time datetime="{{ entry.sent_dt.isoformat() }}">
+                {{- entry.sent_dt|format_human_date -}}
+            </time>
+        </div>
+        {% if not legacy %}
+            <div class="content">
+                <div class="metadata">
+                    <div class="flex">
+                        {%- trans recipients=entry.recipients|sort|join(', ') -%}
+                            <strong>To:</strong> <span class="recipients">{{ recipients }}</span>
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/abstract/notification_log.html]: `To` or `Recipients`? Since it's not only To but also CC

File: indico/htdocs/sass/modules/abstracts/_notification_logs.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        time {
+            color: $dark-gray;
+        }
+    }
+
+    .log-entry > .metadata:not(.legacy) {
+        cursor: pointer;
+
+        &:hover,
+        &.open {
```
Comment (ThiefMaster) [inline indico/htdocs/sass/modules/abstracts/_notification_logs.scss]: is this really more readable than having it in one line? ;x

Comment (OmeGak) [inline indico/htdocs/sass/modules/abstracts/_notification_logs.scss]: Yes.
